src/coreclr/scripts/emitUnitTests.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
Thanks for doing this @a74nh . Perhaps, we should have a python script instead so it can be used on all platforms.

Reply:
That should be possible to just convert across. this might be a good time to try the Arm64 Windows box we've got.

Reply:
yes, but IMO having 2 copies for different platform, makes us remember to make changes on both the scripts and soon one might get outdated with respect to other. 

> this might be a good time to try the Arm64 Windows box we've got

with `python`, you might not even have to do it ;)